react/features/toolbar/actions.native.js

```diff
@@ -0,0 +1,263 @@
+ * @param {boolean} enabled - Indicates if the button should be enabled
+ * or disabled.
+ * @returns {Function}
+ */
+export function setAudioIconEnabled(enabled: boolean = false): Function {
+    return (dispatch: Dispatch<*>) => {
+        const i18nKey = enabled ? 'mute' : 'micDisabled';
+        const i18n = `[content]toolbar.${i18nKey}`;
+        const button = {
+            enabled,
+            i18n,
+            toggled: !enabled
+        };
+
+        dispatch(setToolbarButton('microphone', button));
+    };
+}
+
+/**
+ * Signals that toolbar subject slide in value should be changed.
+ *
+ * @param {boolean} subjectSlideIn - Flag showing whether subject is shown.
+ * @returns {{
+ *     type: SET_SUBJECT_SLIDE_IN,
+ *     subjectSlideIn: boolean
+ * }}
+ */
+export function setSubjectSlideIn(subjectSlideIn: boolean): Object {
+    return {
+        type: SET_SUBJECT_SLIDE_IN,
+        subjectSlideIn
+    };
+}
+
+/**
+ * Signals that value of the button specified by key should be changed.
+ *
+ * @param {string} buttonName - Button key.
+ * @param {Object} button - Button object.
+ * @returns {{
+ *     type: SET_TOOLBAR_BUTTON,
+ *     buttonName: string,
+ *     button: Object
+ * }}
+ */
+export function setToolbarButton(buttonName: string, button: Object): Object {
+    return {
+        type: SET_TOOLBAR_BUTTON,
+        buttonName,
+        button
+    };
+}
+
+/**
+ * Signals that toolbar is hovered value should be changed.
+ *
+ * @param {boolean} isHovered - Flag showing whether toolbar is hovered.
+ * @returns {{
+ *     type: SET_TOOLBAR_IS_HOVERED,
+ *     isHovered: boolean
+ * }}
+ */
+export function setToolbarIsHovered(isHovered: boolean): Object {
+    return {
+        type: SET_TOOLBAR_IS_HOVERED,
+        isHovered
+    };
+}
+
+/**
+ * Dispatches an action which sets new timeout and clears the previous one.
+ *
+ * @param {Function} handler - Function to be invoked after the timeout.
+ * @param {number} toolbarTimeout - Delay.
+ * @returns {{
+ *      type: SET_TOOLBAR_TIMEOUT,
+ *      handler: Function,
+ *      toolbarTimeout: number
+ * }}
+ */
+export function setToolbarTimeout(handler: Function,
+        toolbarTimeout: number): Object {
+    return {
+        type: SET_TOOLBAR_TIMEOUT,
+        handler,
+        toolbarTimeout
+    };
+}
+
+/**
+ * Dispatches an action which sets new toolbar timeout value.
+ *
+ * @param {number} toolbarTimeout - Delay.
+ * @returns {{
+ *      type: SET_TOOLBAR_TIMEOUT_NUMBER,
+ *      toolbarTimeout: number
+ * }}
+ */
+export function setToolbarTimeoutNumber(toolbarTimeout: number): Object {
+    return {
+        type: SET_TOOLBAR_TIMEOUT_NUMBER,
+        toolbarTimeout
+    };
+}
+
+/**
+ * Signals that toolbar visibility value should be changed.
+ *
+ * @param {boolean} visible - Value to be set in redux store.
+ * @returns {{
+ *     type: SET_TOOLBAR_VISIBILITY,
+ *     visible: boolean
+ * }}
+ */
+export function setToolbarVisibility(visible: boolean): Object {
```

Comment:
@BeatC, the names `setToolbarVisibility`, `_onToolbarVisibilityChange`, and `isVisible` do not comply with our coding style which requires 1 name per abstraction. Since the property is called `visible`, a respective function argument (e.g. `_onToolbarVisibilityChange`) should be called `visible` as well, not `isVisible`. Respectively, the function should be called `setToolbarVisible` and, generally, `_onToolbarVisibleChanged` (but in your case the whole name `_onToolbarVisibilityChange` is inappropriate because it's not a handler which responds to changes in the value of a property `visible` of a `Toolbar`). I've taken care of some of the violations of the coding style but you're really expect to be taking care of these (especially since I've repeated the rule in question to you multiple times).